docs/modules/ROOT/pages/references/architecture/keycloak-data-model.adoc

```diff
@@ -0,0 +1,15 @@
+= Keycloak data model
+
+:apub: APPUiO Public 2.0
```

Comment:
What's the reason for that as it is duplicated in each page anyways? Wouldn't it make more sense to have this configure globally for all pages? Afaik OpenShift does it that way for the OpenShift documentation, but I don't know how.

Reply:
This attribute is now defined globally in `docs/antora.yml`.